CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * `ui.Chat()` now correctly handles new `ollama.chat()` return value introduced in `ollama` v0.4. (#1787)
 
+### Changes
+
+* The `shiny.ui.Chat()` class no longer has a `.ui()` method. This method
+was never intended to be used in Shiny Core (in that case, use `shiny.ui.chat_ui()`) to create the UI element. Note that the `shiny.express.ui.Chat()`
+class still has a `.ui()` method. (#1840)
```

Comment:
This could use another pass for clarity, e.g. bringing "In Shiny Core" to the forefront since we use `ui.Chat()` in most places (`shiny.ui` == Shiny Core is likely to be missed).